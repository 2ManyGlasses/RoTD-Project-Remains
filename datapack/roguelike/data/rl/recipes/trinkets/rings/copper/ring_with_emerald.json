{
    "type": "minecraft:crafting_shaped",
    "category": "equipment",
    "pattern": [
      "#M#",
      "# #",
      "###"
    ],
    "key": {
      "#": {
        "item": "minecraft:copper_ingot"
      },
      "M": {
        "item": "minecraft:emerald"
      }
    },
    "result": {
      "id": "minecraft:snort_pottery_sherd",
      "count": 1,
      "components": {
        "minecraft:custom_name": "{\"translate\":\"rl.item.trinkets.ring.copper.emerald\", \"italic\":false, \"color\":\"aqua\"}",
        "minecraft:lore": [
          "{\"translate\":\"rl.item.type.trinket\", \"color\":\"dark_aqua\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.copper.emerald.desc\", \"color\":\"green\",\"italic\":false}",
          "{\"text\":\"***\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.copper.emerald.lore1\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.copper.emerald.lore2\", \"color\":\"dark_gray\",\"italic\":false}"

        ],
        "minecraft:custom_model_data": 3,
        "minecraft:max_stack_size" : 1,
        "minecraft:custom_data": {
            "isTrinket":1,
            "ringEmerald":1
        }
      }
    }
  }
  