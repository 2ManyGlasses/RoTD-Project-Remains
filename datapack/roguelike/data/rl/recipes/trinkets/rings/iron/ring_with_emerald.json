{
    "type": "minecraft:crafting_shaped",
    "category": "equipment",
    "pattern": [
      "#M#",
      "# #",
      "###"
    ],
    "key": {
      "#": {
        "item": "minecraft:iron_ingot"
      },
      "M": {
        "item": "minecraft:emerald"
      }
    },
    "result": {
      "id": "minecraft:shelter_pottery_sherd",
      "count": 1,
      "components": {
        "minecraft:custom_name": "{\"translate\":\"rl.item.trinkets.ring.iron.emerald\", \"italic\":false, \"color\":\"aqua\"}",
        "minecraft:lore": [
          "{\"translate\":\"rl.item.type.trinket\", \"color\":\"dark_aqua\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.iron.emerald.desc\", \"color\":\"green\",\"italic\":false}",
          "{\"text\":\"***\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.iron.emerald.lore1\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.iron.emerald.lore2\", \"color\":\"dark_gray\",\"italic\":false}"

        ],
        "minecraft:custom_model_data": 3,
        "minecraft:max_stack_size" : 1,
        "minecraft:custom_data": {
            "isTrinket":1,
            "ringEmerald":2
        }
      }
    }
  }
  