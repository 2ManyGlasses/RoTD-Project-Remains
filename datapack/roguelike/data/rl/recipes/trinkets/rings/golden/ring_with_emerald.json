{
    "type": "minecraft:crafting_shaped",
    "category": "equipment",
    "pattern": [
      "#M#",
      "# #",
      "###"
    ],
    "key": {
      "#": {
        "item": "minecraft:gold_ingot"
      },
      "M": {
        "item": "minecraft:emerald"
      }
    },
    "result": {
      "id": "minecraft:heartbreak_pottery_sherd",
      "count": 1,
      "components": {
        "minecraft:custom_name": "{\"translate\":\"rl.item.trinkets.ring.golden.emerald\", \"italic\":false, \"color\":\"aqua\"}",
        "minecraft:lore": [
          "{\"translate\":\"rl.item.type.trinket\", \"color\":\"dark_aqua\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.golden.emerald.desc\", \"color\":\"green\",\"italic\":false}",
          "{\"text\":\"***\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.golden.emerald.lore1\", \"color\":\"dark_gray\",\"italic\":false}",
          "{\"translate\":\"rl.item.trinkets.ring.golden.emerald.lore2\", \"color\":\"dark_gray\",\"italic\":false}"

        ],
        "minecraft:custom_model_data": 3,
        "minecraft:max_stack_size" : 1,
        "minecraft:custom_data": {
            "isTrinket":1,
            "ringEmerald":3
        }
      }
    }
  }
  